tests/unit/components/tree/CollapsibleTree.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { mount, Wrapper } from '@vue/test-utils'
```

Comment:
Needs more coverage? Or is it not worth the work?

Reply:
Looked at it more closely and the branch that is not covered is really straight forward and not worth the effort to actually test.